pkg/proxy/endpoints.go

```diff
@@ -34,6 +34,8 @@ import (
 	utilnet "k8s.io/utils/net"
 )
 
+const isHeadlessServiceAnnotation = "service.kubernetes.io/headless"
```

Comment:
Can we make this a const somewhere?

Reply:
Yes we can! I was looking for a better place to this const, as this is used in two different places of the code (controller and proxy) but couldn't find myself with a good solution that doesn't touches other critical parts of the code. 

Maybe, is there somewhere in Kubernetes Code a place (like some sort of types.go) with all the constants? Otherwise, I could import  the controller package into kube-proxy (don't think it's a good idea) or into apimachinery/pkg/types and import it into the endpoints_controller.go code.

Reply:
pkg/proxy/apis/well_known_labels.go

Reply:
OK, got it. About the same annotation into pkg/controller/endpoint/endpoint_controller.go, should I declare it again (as an unexported constant) or import the pkg/proxy/apis into it?

My opinion about this is to redeclare it to keep the code from controller decoupled from proxy, but I may be wrong.

Reply:
How about staging/src/k8s.io/api/core/v1/well_known_labels.go instead?

Reply:
Changed here